src/extension.ts

```diff
@@ -354,6 +363,47 @@ export function activate(context: vscode.ExtensionContext) {
     }));
 
     //
+    // Import of a GLB file and writing out its various chunks.
+    //
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbOpen', () => {
+        const options: vscode.OpenDialogOptions = {
+             canSelectMany: false,
+             openLabel: 'Import',
+             filters: {
+                'Binary glTF': ['glb'],
+                'All files': ['*']
+            }
+        };
+
+        vscode.window.showOpenDialog(options).then(fileUri => {
+            if (fileUri && fileUri[0]) {
+                try {
+                    GlbImport.load(fileUri[0].fsPath);
+                } catch (ex) {
+                    vscode.window.showErrorMessage(ex.toString());
+                }
+            }
+        });
+    }));
+
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbFile', (fileUri) => {
+
+        if (typeof fileUri == 'undefined' || !(fileUri instanceof vscode.Uri)) {
+            if (vscode.window.activeTextEditor === undefined) {
```

Comment:
I'm struggling with this one.  It looks like `importGlbFile` maybe is looking for a GLB file open in the active editor?  If so, it fails, as `fileUri` and `activeTextEditor` are both `undefined` on my machine when a GLB is the active editor window.  When a GLTF file is active, things get worse, as the code below this attempts to read in the plain JSON as a GLB, and fails to find the correct magic number.

Reply:
Maybe we don't need `importGlbFile`, only `importGlbOpen`?

Reply:
This I've been testing with the explorer bar... I haven't got it to work with the opened document.

Reply:
Doh, I forgot to test with explorer bar.  Can this code auto-detect when the command was invoked from explorer vs. when it was invoked some other way?  If nothing else, look to see if `fileUri` actually ends with `.glb`, otherwise an open dialog will be needed.  That way the two import commands could be combined into one.